### ISSUE TITLE
Connection - apiURL, authUrl, tokenUrl are fixed to .nl Endpoints.

### ISSUE THREAD

**Issue by xewl** [picqer/exact-php-client#25]
There's also no methods available to overwrite these values.
```php
    /**
     * @var string
     */
    private $apiUrl = 'https://start.exactonline.nl/api/v1';

    /**
     * @var string
     */
    private $authUrl = 'https://start.exactonline.nl/api/oauth2/auth';

    /**
     * @var string
     */
    private $tokenUrl = 'https://start.exactonline.nl/api/oauth2/token';
```

**Comment by jensdeschrijver** [comment]
Same problem here!

**Comment by casperbakker** [comment]
Did not know there are other endpoints as well. We will add methods to override these. 

What are other available endpoints, for example?

**Comment by xewl** [comment]
https://developers.exactonline.com/#Exact%20Online%20sites.html

The docs state the following countries/tld's are available:

The Netherlands: https://start.exactonline.nl
Belgium: https://start.exactonline.be
Germany: https://start.exactonline.de
United Kingdom: https://start.exactonline.co.uk
United States of America: https://start.exactonline.com

If you use the wrong tld, the client_id won't be found, thus it'll end up in an error saying "Client not found".

**Comment by casperbakker** [comment]
Thanks @xewl, just added it as pull request #26 to add this possibility. I will leave it to @stephangroen to merge it if I did it correctly.

**Comment by xewl** [comment]
You're boss :+1:

**Comment by stephangroen** [comment]
Merged and tagged release v2.2.0 :)
